node/MIGRATION.md

```diff
@@ -42,14 +42,23 @@ chan.handler.register('my-endpoint', function (req, res) {
 Before:
 ```
 chan
-    .send(options, a1, a2, a3, cb);
+    .send(options, a1, a2, a3, function (err, res1, res2) {
+        // CallResponse error is in `err`
+        // CallResponse ok is in `res1`, `res2`
+    });
 ```
 
 After (simple):
 ```
 chan
     .request(options)
-    .send(a1, a2, a3, cb);
+    .send(a1, a2, a3, function (err, res) {
+        // CallResponse is in `res.arg2`, `res.arg3`
+        // res has an `.ok` field that is tells whether it is
+        // an error or not.
```

Comment:
Perhaps explicate that error means application error in this case, and that the "err" argument implies an error carried by an Error frame indicating a failure to deliver the request.